rpc/api/debug_js.go

```diff
@@ -25,37 +25,37 @@ web3._extend({
 			name: 'printBlock',
 			call: 'debug_printBlock',
 			params: 1,
-			inputFormatter: [web3._extend.formatters.inputBlockNumberFormatter]
+			inputFormatter: [null]
```

Comment:
Though as it's only the debug package which probably noone uses, it's not *that* sensitive.